src/org/loklak/api/search/TimeAndDateService.java

```diff
@@ -72,4 +81,34 @@ public static SusiThought timeAndDate() {
 		result.setData(jsonArray);
 		return result;
 	}
+
+    private static Date timeToUTC(Date date) {
+
+        SimpleDateFormat dateFormatGmt = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+        dateFormatGmt.setTimeZone(TimeZone.getTimeZone("GMT"));
+
+        //Local time zone   
+        SimpleDateFormat dateFormatLocal = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+
+        try {
+        //Time in GMT
+        date = dateFormatLocal.parse(dateFormatGmt.format(date));
```

Comment:
The codacy issue is because you reassigned this object here. This check ensures that you do not accidentally modify an object in a function.

Using a new variable like `newDate` would fix the issue.